Week-4/4.1/index.js

```diff
@@ -0,0 +1,46 @@
+// create a custom promise in javascript
+// Create a customPromise
+class CustomPromise {
+    constructor(executor) {
+        this.executor = executor;
+        this.promise = new Promise(executor);
```

Comment:
this defeats the purpose of creating a promise polyfill, please go through the lectures again.